src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -303,17 +303,16 @@ static constexpr Property<Priority> model_priority{"MODEL_PRIORITY"};
  * @ingroup ov_runtime_cpp_prop_api
  */
 enum class PerformanceMode {
-    UNDEFINED = -1,             //!<  Undefined value, performance setting may vary from device to device
```

Comment:
in general, all OpenVINO plugins are supposed to operate in latency mode. I've made UNDEFINED to stick to LATENCY, it seems a backward compatible change. 

But it's still allowed to return tput by default. The global idea is to remove UNDEFINED, since it does not bring any value.